charts/posit-chronicle/LICENSE

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2023 Posit PBC
```

Comment:
Oh I check one PR out and I'm just full of nitpicks.  It's Posit Software PBC

Reply:
Nice catch! We've had this incorrectly copyrighted to `Posit PBC` in the private chart repo too 😐